ext/odbc/tests/odbc_fetch_array_001.phpt

```diff
@@ -0,0 +1,46 @@
+--TEST--
+odbc_fetch_array(): Getting data from query
+--EXTENSIONS--
+odbc
+--SKIPIF--
+<?php include 'skipif.inc'; ?>
+--FILE--
+<?php
+
+include 'config.inc';
+
+$conn = odbc_connect($dsn, $user, $pass);
+
+odbc_exec($conn, 'CREATE TABLE fetch_array (foo INT)');
+odbc_exec($conn, 'INSERT INTO fetch_array VALUES (1)');
+odbc_exec($conn, 'INSERT INTO fetch_array VALUES (2)');
+odbc_exec($conn, 'INSERT INTO fetch_array VALUES (3)');
```

Comment:
I inserted 3 items because the first one is retrieved with the row number specified (`odbc_fetch_array($res, 1)`), the 2nd one by retrieving the next one (`odbc_fetch_array($res)`), and the 3rd one by specifying the row number again to make sure that automatic incrementing works (`odbc_fetch_array($res, 3)`). The 4th one is to check an error condition.